ci-operator/step-registry/ipi/deprovision/vsphere/ipi-deprovision-vsphere-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
+tfvars_path=/var/run/secrets/ci.openshift.io/cluster-profile/secret.auto.tfvars
+base_domain=origin-ci-int-aws.dev.rhcloud.com
```

Comment:
I'd rather extract this from a mounted secret (`/var/run/secrets/...`), instead of hard-coding it in two separate steps.

Reply:
What about a BASE_DOMAIN envvar set in the job?